<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>661</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>661</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="consoleOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>761</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="NewWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PCAP LIST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DELETETEMP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>网路介面卡</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>筛网</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>761</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ADDITEM">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ADDITEM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DELETEITEM">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DELETEITEM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SETITEM">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SETITEM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>761</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>###########################################        LOG        ##########################################</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>94</y>
      <width>761</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>########################################        FILTER ITEM        ########################################</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
